terra/asset.json

```diff
@@ -1117,7 +1117,15 @@
     "type": "cw20",
     "icon": "https://raw.githubusercontent.com/lioncubdao/logo/main/Screenshot%202023-04-10%20115645.png"
   },
-  {
+    {
+    "id": "terra1gwrz9xzhqsygyr5asrgyq3pu0ewpn00mv2zenu86yvx2nlwpe8lqppv584",
```

Comment:
@lioncubdao Hey Lion DAO team! There's already an existing entry for the `BLUE` token with the same `id` (around line 2001), could you update the necessary fields for that entry instead?

Reply:
we have updated the existing entry and deleted the other one, thank you :)